hashes/zkevm-keccak/src/keccak_packed_multi.rs

```diff
@@ -287,6 +288,13 @@ impl<F: FieldExt> CellManager<F> {
         } else {
             assert!(column_idx == self.columns.len());
             let advice = meta.advice_column();
+            if let Ok(col_to_enable) = var("COL_TO_ENABLE")
```

Comment:
This part of the code is only being called when we need an extra col, otherwise we don't hit the `else` branch. So it won't be called enough times to affect the performance. 

I agree that ideally we should store the advice column for which we need to enable equality in the CellManager though.